cmd/lakefs/cmd/run.go

```diff
@@ -83,6 +84,15 @@ var runCmd = &cobra.Command{
 			logger.WithError(err).Fatal("Failed to create block adapter")
 		}
 
+		//  Migrate old MVCC - if cataloger type is not set,
+		//  warn the user in case migrate didn't run and there are MVCC repositories
+		if cfg.GetCatalogerType() == "" {
+			migrationRequired := catalogmigrate.CheckMigrationRequired(dbPool)
+			if migrationRequired {
+				logger.Fatal("Migration of old data is required! (lakefs migrate db)")
```

Comment:
We need to print out a very nice message here:

_Starting version 0.30.0, lakeFS handles your committed metadata in a new way, which is more robust and has better performance. To move your existing data, you will need to run the following upgrade command:
...
If you want to start over, ignoring your existing data, yoou need to explicitly state this in your lakeFS configuration file. To do so, add the following to your configuration:
```
cataloger:
  type: rocks
```_

I think we also need to add this to the documentation